great_expectations/cli/datasource.py

```diff
@@ -583,7 +583,7 @@ def _collect_postgres_credentials(default_credentials=None):
     if default_credentials is None:
         default_credentials = {}
 
-    credentials = {"drivername": "postgres"}
+    credentials = {"drivername": "postgresql"}
```

Comment:
yikes.

Reply:
FWIW, it's not awful -- postgres still works for now but is deprecated.